contracts/ethereum/src/core/node-codec/ScaleCodec.sol

```diff
@@ -1,9 +1,7 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.17;
 
-import "../../interfaces/ICodec.sol";
-
-contract ScaleCodec is ICodec {
+contract ScaleCodec {
```

Comment:
why not referencing the interface any more?

Reply:
not needed anymore. It's in use by NodeCodec contract though.